script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1C.txt

```diff
@@ -66,26 +66,26 @@ Arcueid sits upright on the bed.
 She's in full-on chatter mode.
 "―――Geez."
 I don't have a choice.
-I've got some questions I need to ask her as well, so I'll just talk to her for a bit.
+I've got some questions I need to ask her as well, so I'll just talk with her for a bit.
 <Page20>
-"Arcueid. Can I ask you about the hotel from yesterday?"
+"Arcueid. Can I ask you about what happened at the hotel yesterday?"
 "...That's right. Pretty natural you'd want to talk about that after what you saw."
 <Page21>
 "That guy from yesterday... I believe he called himself Vlov――what the hell is he?
-I understand that he's a vampire. That grandiose outfit and such makes him look much more vampire-like than you after all. But―――"
+I understand that he's a vampire. That grandiose outfit makes him look much more vampire-like than you after all. But―――"
 <Page22>
-That blue flame. What was that flame that filled the hallway and burned humans but left everything else intact? That was too far from my image of a vampire.
+That blue flame. What was that flame that filled the hallway and burned humans but left everything else intact? That didn't fit with how I pictured vampires at all.
 <Page23>
 "Hmph. What do you mean he's more 'vampire-like' than me?
 Though... it's just like you said.
-His body isn't decrepit and his Principle is fairly small-scale. He's the sort of 'orthodox' vampire that humans would imagine.
+His body isn't falling apart and his Principle is fairly small-scale. He's the sort of 'orthodox' vampire that humans would imagine.//Principle / scale here are 原理/規模
```

Comment:
He's the sort of 'orthodox' vampire that humans think of.

Reply:
Minor quibble with 'think of' is that it makes the concept of a vampire like that seem like something that _exists_ and people are familiar with / can refer back to (and obviously, the _concept_ does), but the real deal image of Vlov is something that most people can only picture / imagine less substantially. Very nitpicky though.

Reply:
After reading it again, I think my issue is with 'would'

He's the sort of 'orthodox' vampire that humans imagine.

Reply:
After reading it again, I think my issue is with 'would'

He's the sort of 'orthodox' vampire that humans imagine.

Reply:
I like "that humans imagine."